backend/backend.ts

```diff
@@ -18,9 +21,21 @@ const PORT = 3001;
 // Setup Express
 // *************************************************
 
+// Configure rate limiter: maximum of 100 requests per 5 minutes (10 a minute)
```

Comment:
Do we have anything to track when the rate limit is reached?

Reply:
It does this internally, I dare not question how, but I have tested it and it indeed does work.

There's actually now two layers of rate limiting:
1. Cloudflare rate limits site wide, restricted to 20reqs/10s
2. API app rate limits, using the two ones configured in this file.

I chose to be more stringent on the share one as there's possibility for abuse, whereas the save and load endpoints require user authentication, which could still be botted but at least it's more difficult to do so.